fastlane/Fastfile

```diff
@@ -78,6 +79,71 @@ lane :release do |options|
   github_release(version: version_number)
 end
 
+desc "Creates GitHub snapshot pre-release and publishes with the 'snapshot' tag"
+lane :snapshot_release do
+  stable_version   = current_version_number
+  snapshot_version = "#{stable_version}-SNAPSHOT.#{Time.now.strftime('%Y%m%d%H%M')}"
+
+  UI.message("Creating snapshot release #{snapshot_version}...")
+
+  Dir.chdir(get_root_folder) do
+    sh("npm version #{snapshot_version} --no-git-tag-version")
+    sh("npm publish --tag snapshot")
+    sh("npm version #{stable_version} --no-git-tag-version")
+  end
+
+  add_git_tag(tag: snapshot_version, message: "Snapshot #{snapshot_version}")
```

Comment:
Also, how about prefixing all snapshot versions with something? Maybe starting with `snapshot-X.Y.Z`? Mostly so they are grouped together in github and the stable releases stay at the top.